browser-support-molecule.css

```diff
@@ -552,6 +554,7 @@ body {
     -webkit-flex-wrap: wrap;
     -ms-flex-wrap: wrap;
     flex-wrap: wrap;
+    height: 48px;
```

Comment:
For responsive: Line 557 messes up the flex-boxes during smaller viewport. You need to remove it.